cmd/apiserver/app/server/run_server.go

```diff
@@ -183,7 +183,7 @@ func (c checkEtcdConnectable) Check(_ *http.Request) error {
 	serverReachable, err := preflight.EtcdConnection{ServerList: c.ServerList}.CheckEtcdServers()
 
 	if err != nil {
-		glog.Error("etcd checker failed with err: %v", err)
+		glog.Errorf("etcd checker failed with err: %v", err)
```

Comment:
@MHBauer fix for your change in https://github.com/kubernetes-incubator/service-catalog/pull/1039
not sure how did it pass through Travis verification build:
```
Running golint and go vet:
docker run --rm -v /Users/nislamov/go/src/github.com/kubernetes-incubator/service-catalog:/go/src/github.com/kubernetes-incubator/service-catalog -v /Users/nislamov/go/src/github.com/kubernetes-incubator/service-catalog/.pkg:/go/pkg scbuildimage go vet ./cmd/... ./contrib/... ./pkg/... ./plugin/... ./test/...
cmd/apiserver/app/server/run_server.go:186: possible formatting directive in Error call
exit status 1
make: *** [verify] Error 1
```